SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -524,6 +524,11 @@ void USpatialNetDriver::NotifyActorDestroyed(AActor* ThisActor, bool IsSeamlessT
 	RenamedStartupActors.Remove(ThisActor->GetFName());
 }
 
+const FString USpatialNetDriver::GetUniqueIdentifier() const
+{
+	return Connection->GetWorkerId();
```

Comment:
Ensure Connection is valid before using.

Reply:
Added check above

Reply:
How about -
if (Connection != nullptr)
{
return etc.
}
return Super::GetUniqueIdentified();

Reply:
Aye sounds better